plugin-server/src/types.ts

```diff
@@ -857,6 +857,7 @@ export interface RawSessionRecordingEvent {
     session_id: string
     window_id: string
     snapshot_data: string
+    events_summary: string
```

Comment:
Have you tried this to make ingestion work?

```suggestion
    events_summary: string[]
```